pyccl/halos/halo_model.py

```diff
@@ -89,6 +91,12 @@ def __init__(self, cosmo, massfunc, hbias, mass_def,
         self._a_current_mf = -1
         self._a_current_bf = -1
 
+    @deprecate_attr(pairs=[("mass_function", "_massfunc"),
+                           ("halo_bias", "_hbias"),
+                           ("mass_def", "_mdef")])
```

Comment:
these ones, I meant

Reply:
OK this is addressed above, resolving it.